bundles/org.eclipse.jface.text/src/org/eclipse/jface/internal/text/StickyHoverManager.java

```diff
@@ -142,6 +144,14 @@ public void stop() {
 			if (fInformationControlToClose != null)
 				fInformationControlToClose.removeFocusListener(this);
 
+			// Was added to fix the bug #2534
```

Comment:
Nice to have this, but it doesn't help the reader to understand the code (this info is available also by git blame). Better is to say **why** we manually touch the shell here.